.github/workflows/release.yml

```diff
@@ -272,9 +272,10 @@ jobs:
       - name: Create portable coordinator resource definitions
         run: |
           mkdir -p workspace deployment
-          nix run .#scripts.write-coordinator-yaml -- "${coordinatorImgTagged}" > workspace/coordinator.yml
-          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --namespace kube-system runtime > workspace/runtime.yml
-          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --add-load-balancers emojivoto-sm-ingress > deployment/emojivoto-demo.yml
+          # TODO(msanft): Release for all platforms once stable
+          nix run .#scripts.write-coordinator-yaml -- "${coordinatorImgTagged}" AKS-CLH-SNP > workspace/coordinator.yml
+          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --namespace kube-system --platform AKS-CLH-SNP runtime > workspace/runtime.yml
+          nix shell .#contrast --command resourcegen --image-replacements ./image-replacements.txt --add-load-balancers --platform AKS-CLH-SNP emojivoto-sm-ingress > deployment/emojivoto-demo.yml
```

Comment:
Is the `--platform` flag already implemented in `resourcegen`?

Reply:
This will be implemented in a follow-up, where I implement the actual logic in the node-installer and the generation for its deployment files. So theoretically, we could leave it out of this PR for now.